include/openthread.h

```diff
@@ -1188,7 +1268,7 @@ ThreadError otRemoveExternalRoute(otInstance *aInstance, const otIp6Prefix *aPre
  * @sa otAddExternalRoute
  * @sa otRemoveExternalRoute
  */
-ThreadError otSendServerData(otInstance *aInstance);
+OTAPI ThreadError OTCALL otSendServerData(otInstance *aInstance);
 
 /**
  * This function adds a port to the allowed unsecured port list.
```

Comment:
We should likely consider rename them to something like "AssistingPorts"... We use them for commissioning new devices.